packages/shared/src/logger.ts

```diff
@@ -51,13 +51,14 @@ class Logger {
     this.sessionId = randomUUID();
   }
 
+  // This should be called with the name once at the entry point.
+  // For example, with the Playwright plugin, it is called in the Reporter interface constructor.
   initialize(app: string, version: string | undefined) {
```

Comment:
this has to be called somewhere in the old CLI too - let's settle on the other details first though